frontend/src/components/ExternalDb.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="section extended-section">
+    <div v-if="externalDb" class="container is-fullhd">
+      <h3 class="title">
+        {{ externalDb.dbName }} {{ components[0].componentType }} {{ externalDb.externalId }}
+      </h3>
+      <h4 class="subtitle">
```

Comment:
To avoid having this text extra large, this change would match the style of the rest of our pages:
```suggestion
      <p>
```

Reply:
Sounds good! @inghylt also suggested something along these lines with an additional change.

Reply:
![image](https://user-images.githubusercontent.com/32480198/119824141-ded99b00-bef5-11eb-873f-b7ca26390ba4.png)
Just replacing it with `<p>` made the spacing between the elements quite weird IMO so I played around with the margins a bit. Does this look good to you?

Reply:
@e0 haha yeah I was referring to the spacing between the elements. Thanks for input :+1: